api.md

```diff
@@ -469,7 +531,77 @@ CREATE INDEX ON conditions(time, location) USING brin
 ```
 
 ---
+## detach_data_node() [](detach_data_node)
+
+Detach a data node from one hypertable or from all hypertables.
 
+This function is typically used if a data node should no longer be
+used by a hypertable. Then it need to be removed from all hypertables
+that use it.
+
+Another reason for detaching a data node from a hypertable is that
+you want to have fewer partitions for that hypertable compared to
+other hypertables in the database. This function *MUST* be run on an access
+node.
+
+#### Required Arguments [](detach_data_node-required-arguments)
+
+| Name        | Description                       |
+| `node_name` | Local name of data node to detach |
+
+##### Parameter Description
+
+* `node_name` - Local name of the data node to detach from the
+  hypertable.
+
+#### Optional Arguments [](detach_data_node-optional-arguments)
+
+| Name         | Description                            |
+| `hypertable` | Name of hypertable to detach node from |
+| `force`      | Force detach of the data node          |
+
+##### Parameter Description
+
+* `hypertable` - The name of the hypertable where the data node should
+  be attached. If NULL, the data node will be detached from all
+  hypertables.
+
+* `force` - Force detaching the node even if that means that the
+  replication factor is reduced below what was required. Note that it
+  will never be allowed to reduce the replication factor below 1 since
+  that would cause data loss.
+
+#### Returns
+
+| Column               | Description                              |
+| `hypertable_id`      | Hypertable id of the modified hypertable |
+| `node_hypertable_id` | Hypertable id on the remote data node    |
+| `node_name`          | Local name of the attached data node     |
+
+#### Errors
+
+If the data node contains chunks that are not replicated on other data
+nodes or if detaching the data node would result in a replication factor
+below what was configured, an error will be given. Detaching a data node is
```

Comment:
Split into list format:

"Detaching a data node is not permitted:
 - If it would result in a data loss for the hypertable
 - If it would result in under-replicated chunks for the distributed hypertable (without the `force` argument)"

Reply:
Done